src/http-client.js

```diff
@@ -254,6 +254,13 @@ const orderOco = (privCall, payload = {}, url) => {
   )
 }
 
+const exchangeInfo = (pubCall, payload) => {
```

Comment:
this whole function is not only hacky and un-prettied, but also introduces new unexpected behavior which makes me question your motives.

Reply:
Can you elaborate what you don't like about it? I basically copied code from other methods and adapted it for this endpoint. If you think it introduces unexpected behavior, why not supply a unit test that this function fails? Then I can surely fix it.

Reply:
Since `checkParams` can only check for mandatory parameters and `symbol` is an optional parameter, I don't see a better way to implement it.